personalization-service/src/test/java/de/muenchen/dbs/personalization/checklist/ChecklistIntegrationTest.java

```diff
@@ -3,26 +3,16 @@
 import static de.muenchen.dbs.personalization.checklist.ChecklistTestHelper.createTestChecklist;
 import static org.hamcrest.Matchers.hasSize;
 import static org.hamcrest.Matchers.is;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
 
 import de.muenchen.dbs.personalization.IntegrationTestBase;
 import de.muenchen.dbs.personalization.TestConstants;
-import de.muenchen.dbs.personalization.checklist.domain.Checklist;
-import de.muenchen.dbs.personalization.checklist.domain.ChecklistCreateDTO;
-import de.muenchen.dbs.personalization.checklist.domain.ChecklistItem;
-import de.muenchen.dbs.personalization.checklist.domain.ChecklistMapper;
-import de.muenchen.dbs.personalization.checklist.domain.ChecklistUpdateDTO;
+import de.muenchen.dbs.personalization.checklist.domain.*;
```

Comment:
Please no asterisk imports

Reply:
done [2ba6a43](https://github.com/it-at-m/dbs/pull/238/commits/2ba6a4368e2189c9f15dd8e6ff49e76513c1f2df)